c7n/resources/vpc.py

```diff
@@ -246,6 +247,12 @@ def __call__(self, resource):
                         only_found = True
                 if self.only_ports and not only_found:
                     found = True
+            if 'IpRanges' in perm and 'IpRanges' in self.data:
```

Comment:
this check can go into handle_cidrs, also the utils class Cidr with static methods should handle the conversion just pass the string.